### ISSUE TITLE
comm: file 1 is not in sorted order

### ISSUE THREAD

**Issue by MacFlurry** [pi-hole/docker-pi-hole#403]
**In raising this issue, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).
Hello,

doing `docker logs pihole`
I noticed this:
```
comm: file 1 is not in sorted order
comm: file 1 is not in sorted order
comm: file 1 is not in sorted order
comm: file 1 is not in sorted order
comm: file 1 is not in sorted order
comm: file 1 is not in sorted order
```
 How this could be fixed ?

Best regards

**Comment by ail1020** [comment]
I don't believe this issue is specific to the docker version as I also get it even though I'm not using docker. I get this error twice per gravity update. It seems to be specific to blocklists in the "Adblock" format. Here's a sample of my output when running `pihole -g`

>   [✓] Status: Retrieval successful
>   [i] Format: Adblockcomm: file 1 is not in sorted order                                                                                           
>   [✓] Format: Adblock

Try updating gravity and check the output so see if it also shows that error on the "Adblock" formatted files or if you're seeing a different behavior.

**Comment by MacFlurry** [comment]
I don't have Adblock filters
